vcr/types/interface.go

```diff
@@ -64,10 +65,18 @@ type ConceptFinder interface {
 	// It also returns untrusted credentials when allowUntrusted == true
 	Get(conceptName string, allowUntrusted bool, subject string) (concept.Concept, error)
 
-	// Search for matching concepts based upon a query. It returns an empty list if no matches have been found.
+	// SearchConcept returns matching concepts based upon a query. It returns an empty list if no matches have been found.
+	// It also returns untrusted credentials when allowUntrusted == true
+	// a context must be passed to prevent long-running queries
+	SearchConcept(ctx context.Context, conceptName string, allowUntrusted bool, query map[string]string) ([]concept.Concept, error)
```

Comment:
is the `ConceptFinder` something that will be deprecated and replaced by JSON-LD schemas in the future?

Reply:
yup